lib/archethic_web/templates/node/list.html.leex

```diff
@@ -4,30 +4,56 @@
   <div class="column">
     <div class="box">
       <p class="heading">Authorized nodes (<%= Enum.count(@authorized_nodes) %>)</p>
-        <%= for node <- @authorized_nodes do %>
-          <div class="box">
-            <div class="columns is-desktop">
-              <div class="column">
-                  <div>
-                      <p class="heading">Last public key</p>
-                      <%= link to: Routes.live_path(@socket, ArchethicWeb.NodeDetailsLive, Base.encode16(node.last_public_key)) do%>
-                        <span class="is-hidden-mobile"><%= Base.encode16(:binary.part(node.last_public_key, 0, 30)) %>...</span>
-                        <span class="is-hidden-desktop"><%= Base.encode16(:binary.part(node.last_public_key, 0, 13)) %>...</span>
-                      <% end %>
-                  </div>
-              </div>
-              <div class="column">
-                  <div>
-                      <p class="heading">First public key</p>
-                      <%= link to: Routes.live_path(@socket, ArchethicWeb.NodeDetailsLive, Base.encode16(node.last_public_key)) do%>
-                        <span class="is-hidden-mobile"><%= Base.encode16(:binary.part(node.first_public_key, 0, 30)) %>...</span>
-                        <span class="is-hidden-desktop"><%= Base.encode16(:binary.part(node.first_public_key, 0, 13)) %>...</span>
-                      <% end %>
-                  </div>
+      <div class="columns">
+        <div class="column">
+          <article class="message is-warning">
+              <div class="message-body">
+                 Contains the list of nodes which have been authorized during the node shared secrets renewal and will participate in the transaction validation
               </div>
+          </article>
+        </div>
+      </div>
+      <%= for node <- @authorized_nodes do %>
+        <div class="box">
+          <div class="columns is-desktop">
+            <div class="column">
+                <div>
+                    <p class="heading">First public key</p>
+                    <%= link to: Routes.live_path(@socket, ArchethicWeb.NodeDetailsLive, Base.encode16(node.last_public_key)) do%>
+                      <span class="is-hidden-mobile"><%= Base.encode16(:binary.part(node.first_public_key, 0, 30)) %>...</span>
+                      <span class="is-hidden-touch"><%= Base.encode16(:binary.part(node.first_public_key, 0, 13)) %>...</span>
```

Comment:
To be more responsive you should do something like this to avoid double line :
```html
<span class="is-hidden-tablet"><%= Base.encode16(:binary.part(node.first_public_key, 0, 10)) %>...</span>
```

Reply:
done